includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -450,7 +450,7 @@ public function payment_methods_check_environment() {
 	}
 
 	/**
-	 * Adds a notice to the order details page if the subscription payment method has been detached.
+	 * Adds a notice to the order details page if the (active) subscription payment method has been detached.
```

Comment:
I think it would be clearer to express this as follows, because I find it hard to understand what the `(active)` applies to.
```suggestion
	 * Adds a notice to the subscription details page if we are looking at an active subscription and the payment method has been detached.
```